### ISSUE TITLE
audio

### ISSUE THREAD

**Issue by Aman49362** [russelltg/wl-screenrec#66]
is there a way i can record specific audio outputs ?
like an application with my mic

**Comment by russelltg** [comment]
Ah, right now you can only record one audio stream, specified by --audio-device, so turning this issue into a feature request to mix audio streams.

**Comment by russelltg** [comment]
Note, if you use pipewire, this is doable by making a virtual output:

```
pactl load-module module-null-sink media.class=Audio/Sink sink_name=wl-screenrec-input channel_map=stereo
```

Then fire up qwpgraph and setup your graph as desired, something like this: 

![image](https://github.com/russelltg/wl-screenrec/assets/7105082/8cabacc2-172b-4bb4-a2db-e1c0fd1eefa1)

then run 

```
wl-screenrec --audio --audio-device wl-screenrec-input.monitor
```

**Comment by russelltg** [comment]
Yeah, you're going to have to drag-n-drop connections. For the method I outlined above.

Don't mark this as solved, I think it is a reasonable request. I was just posting a workaround for pipewire users.

**Comment by Aman49362** [comment]
Oh got it all i was just confused for what command or application to use for merging this 
i search in the package manager for qwpgraph which you spelled wrong
but no problem i got it working now thanks to you appreciate you work sir
and guess i should mark it as solved just let me know
